cmd/server.go

```diff
@@ -873,6 +879,9 @@ func (s *ServerCmd) setDefaults(c *server.UserConfig) {
 	if c.MarkdownTemplateOverridesDir == "" {
 		c.MarkdownTemplateOverridesDir = DefaultMarkdownTemplateOverridesDir
 	}
+	if !v.IsSet("max-comments-per-command") {
+		c.MaxCommentsPerCommand = DefaultMaxCommentsPerCommand
+	}
```

Comment:
as "0" is an acceptable value with a specific meaning, I check if the flag for the max-comments-per-command is set before setting the default value in the user config.

IMHO, this is how we should do for all flags, because checking for empty strings or 0 values (set by golang by default when unmarshalling into the UserConfig struct) can be misleading.

Reply:
Yeah that makes sense to me. We should do a pass of the other flags at some point